src/main/scala/viper/gobra/frontend/info/implementation/typing/ExprTyping.scala

```diff
@@ -521,6 +525,10 @@ trait ExprTyping extends BaseTyping { this: TypeInfoImpl =>
     case PLength(op) => isExpr(op).out ++ {
       underlyingType(exprType(op)) match {
         case _: ArrayT | _: SliceT | _: GhostSliceT | StringT | _: VariadicT | _: MapT | _: MathMapT => noMessages
+        case ActualPointerT(_: ArrayT)  =>
+          // Go allows getting the length of a pointer to an array, but it does not allow obtaining the
+          // length of a pointer of a type T whose underlying slice is an array.
```

Comment:
```suggestion
          // Go allows getting the length of a pointer to an array, but it does not allow obtaining the
          // length of a pointer of type T, whose underlying type is an array.
```

Reply:
I agree with this suggestion, except for the comma before "whose", given that the clause introduced by whose is supposed to be restrictive. Putting the comma thus changes the meaning of the sentence